server/routerlicious/packages/routerlicious-base/src/test/alfred/api.spec.ts

```diff
@@ -204,12 +204,13 @@ describe("Routerlicious", () => {
 
 				const assertThrottle = async (
 					url: string,
-					token: string | (() => string),
-					body: any,
+					token: string | (() => string) | undefined,
+					body: any | undefined,
```

Comment:
Any stronger typing we can use here than `any`?

Reply:
We _should_ use `unknown` but that is out of scope for this work, IMO. There needs to be a whole different effort put into making all user input `unknown` and then doing type validations on it.